dcf/views.py

```diff
@@ -207,3 +208,30 @@ def dispatch(self, *args, **kwargs):
 class RobotsView(TemplateView):
     template_name = 'dcf/robots.txt'
     content_type = 'text/plain'
+
+
+class ComplaintCreateView(CreateView):
+    model = Complaint
+    fields = ['title', 'text']
+
+    def form_valid(self, form):
+        form.instance.user = self.request.user
+        form.instance.item = Item.objects.get(id=self.kwargs['item_id'])
+        form.save()
+        self.send_email(form)
+        return super(ComplaintCreateView, self).form_valid(form)
+
+    def send_email(self, form):
+        mail_to = [admin[1] for admin in dcf_settings.settings.ADMINS]
+        mail_subject = "Поступила жалоба на объявление"
+        mail_text = str(
+            'Объявление: "{item}"\n' +
+            '{title}\n' +
+            '{text}').format(
+                item=form.instance.item.title,
+                title=form.instance.title, text=form.instance.text)
+        send_mail(mail_subject, mail_text, 'noreply@localhost', mail_to)
```

Comment:
Please use DEFAULT_FROM_EMAIL and that email should be sent to all needs to be sent to all ADMINS